examples/square.rs

```diff
@@ -41,25 +42,37 @@ pub enum MapLayers {
     DenseExtra,
 }
 
+/// Change to change the square tile sizes
+const TILE_SIZE: f32 = 15.0;
+
+/// Change to change the gap between tiles
+const TILE_GAP: f32 = 1.0;
+
 #[derive(Hash, Default, Copy, Clone, Reflect)]
 struct TileData(u8, u8);
 
 // Resource to hold our map entity so we can use it in systems
 #[derive(Resource)]
 pub struct MapEntity(Entity);
 
-fn spawn_map(mut commands: Commands) {
-    let map_size = UVec2::new(500, 500);
-    let max_chunk_size = UVec2::new(100, 100);
+#[derive(Resource)]
+pub struct SquareMeshHandle(Mesh2d);
+
+#[derive(Resource, Default)]
+pub struct ColorHandles(Vec<Handle<ColorMaterial>>);
+
+fn spawn_map(mut commands: Commands, mut meshes: ResMut<Assets<Mesh>>) {
+    let map_size = UVec2::new(5, 5);
```

Comment:
I want to keep the map just big enough that the chunks make sense
```suggestion
    let map_size = UVec2::new(10, 10);
```